src/api/routes/bookmarkRouter.js

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import bookmarkController from '../controllers/bookmarkController';
+import authorization from '../../middlewares/authorization';
+
+const route = new Router();
+
+route.post('/:slug', authorization, bookmarkController.addBookmark);
```

Comment:
@truestbyheart check also if you are receiving an empty object

Reply:
Am not receiving any object @nkpremices I just use the token and the slug